docs/analyst_permissions.md

```diff
@@ -0,0 +1,16 @@
+### Overview of analyst permissions
```

Comment:
@matthieu-foucault I started this but was unsure of what you all wanted me to add here.

Reply:
This looks great. Maybe some technical details can be added, regarding pages access:
 - which pages can be accessed by which user
 - what is the page redirection behaviour
 - a note that client-side routing will not go through the page redirection logic. The other day, I was suggesting that the logo link should use the back-end routing instead of a Nextjs Link. I remember now that this is the reason why

Reply:
Sounds good, I updated the docs.